src/main/groovy/wooga/gradle/unity/UnityPlugin.groovy

```diff
@@ -265,7 +267,8 @@ class UnityPlugin implements Plugin<Project> {
                     }
                 })
 
-                project.artifacts.add(UNITY_PACKAGE_CONFIGURATION_NAME, [file: task.archivePath, builtBy: task])
+                UnityPackageArtifact artifact = UnityPackageArtifact.fromTask(task)
+                configuration.getArtifacts().add(artifact)
```

Comment:
The old code would have worked the same.
```
project.artifacts.add(UNITY_PACKAGE_CONFIGURATION_NAME, artifact)
```

But I guess the paket logic does the same?

Reply:
I thought it might be nice to do it the same way over the different packages